encryption_works.md

```diff
@@ -220,37 +220,19 @@ With the release of OTR 4.0 in September 2012, Pidgin stopped logging OTR conver
+PGP stands for "Pretty Good Privacy," software you can use to encrypt your e-mail messages. GPG is a free, open-source version of this software. The terms PGP and GPG are often used interchangeably. Unfortunately, the process is notoriously hard to use, as seen by Glenn Greenwald [explaining](http://www.huffingtonpost.com/2013/06/10/edward-snowden-glenn-greenwald_n_3416978.html?1370895818) how he could not initially talk to Edward Snowden because PGP is so difficult to set up.
 
-Today, PGP is a company that sells a proprietary encryption program by the same name. [OpenPGP](http://openpgp.org/) is the open protocol that defines how PGP encryption works, and [GnuPG](http://www.gnupg.org/) (GPG for short) is free software, and is 100% compatible with the proprietary version. GPG is much more popular than PGP today because it's free for everyone to download, and cypherpunks trust it more because it's open source. The terms PGP and GPG are often used interchangably.
+If you use PGP encryption, you will have to make some changes about how you use e-mail. For example, if you use PGP on your computer but receive an encrypted e-mail on your phone, you won't be able to decrypt the e-mail and read it until you get back to your computer.
 
-Unfortunately, PGP is notoriously hard to use, as exemplified by [Greenwald explaining how he could not initially talk to Edward Snowden because it was so difficult to set up](http://www.huffingtonpost.com/2013/06/10/edward-snowden-glenn-greenwald_n_3416978.html?1370895818).
+PGP is used in two ways with e-mail: it can encrypt the content of e-mail messages, and it can sign messages, proving that the message the sender sent is the same one you're now reading, and that it wasn't tampered with in transit. (This assumes you trust the public key -- more on that later.)
 
-### Keypairs and Keyrings
+### What's a key? 
 
-As with OTR, each person who wishes to send or receive encrypted email needs to generate their own PGP key, called a keypair. PGP keypairs are split into two parts, the public key and the secret key.
+PGP uses a public key and a private key, both strings of randomly-generated numbers and letters unique to each person. Information about the public key is stored on something called a public key server, which is a little like a phonebook. When people want to e-mail you securely, they can look up your public key and send you an encrypted e-mail. 
 
-If you have someone's public key, you can do two things: **encrypt messages** that can only be decrypted with their secret key, and **verify signatures** that were generated with their secret key. It's safe to give your public key to anyone who wants it. The worst anyone can do with it is encrypt messages that only you can decrypt.
+You can think of your public key and private key like your e-mail address and e-mail passphrase. You probably want your general e-mail address to be public, so that people can contact you -- it's the same with your public key. 
```

Comment:
Great start, and SUPERB analogy!  Please continue, underscoring that the private key should remain private at all costs...